boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7_defconfig

```diff
@@ -1,8 +1,11 @@
 CONFIG_SOC_R8A77951=y
 CONFIG_SOC_SERIES_RCAR_GEN3=y
 CONFIG_BOARD_RCAR_H3ULCB_CR7=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CLOCK_CONTROL_RCAR_CPG_MSSR=y
```

Comment:
Some of the other platforms have something like
select CLOCK_CONTROL_RCAR_CPG_MSSR if CLOCK_CONTROL in the soc Kconfig.soc file instead of defining the specific clock control driver in every board defconfig.

Reply:
indeed it make sense, will fix